### PR TITLE
build.gradle: Use -PtestMinJdk=8 instead of testJdk8

### PR DESCRIPTION
**Pull request by msgilligan**
Rather than use a boolean that runs hardcoded JDK8 tests, use a numeric parameter to set the minimum JDK to test on.

This makes it easier to change our minimum version and it also makes it possible to run the tests on the minimum available JDK on a particular platform or really any JDK you want.

### CODE REVIEW

File: base/build.gradle [hunk 1]
```diff
@@ -38,23 +38,22 @@ test {
 }
 
 def gradleVersionToolchains = GradleVersion.version("6.7")
+// Activate if `testMinJdk` is a integer > 0 in `gradle.properties` or `-PtestMinJdk=true` is on command-line
```
Comment (schildbach) [inline base/build.gradle]: > `-PtestMinJdk=true`

Really? I thought that argument is now an integer?

File: core/build.gradle [hunk 2]
```diff
@@ -85,23 +85,22 @@ test {
 }
 
 def gradleVersionToolchains = GradleVersion.version("6.7")
+// Activate if `testMinJdk` is a integer > 0 in `gradle.properties` or `-PtestMinJdk=true` is on command-line
```
Comment (schildbach) [inline core/build.gradle]: Same here.

File: base/build.gradle (hunk 1 above)
Comment (msgilligan) [reply base/build.gradle]: Good catch. I'll fix the comment.

Comment (msgilligan) [reply base/build.gradle]: Fixed and force-pushed.

File: core/build.gradle (hunk 2 above)
Comment (msgilligan) [reply core/build.gradle]: Fixed and force-pushed.
